openmeetings-web/src/main/java/org/apache/openmeetings/web/room/wb/raw-wb-board.js

```diff
@@ -369,6 +370,52 @@ var Wb = function() {
 					minHeight: 140
 					, minWidth: 255
 				});
+				tools.find('.om-icon.dweet').click(function(){					
+			        dweet = OmUtil.tmpl("#wb-dweet");			        
+			       	dweet[0].id='wb-dweet' + dweetIDCount++;        
+			        dweet[0].style.bottom = '100px';
+			        dweet[0].style.position= 'absolute';
+			        dweet[0].style[(Settings.isRtl ? 'left' : 'right')] = '100px';        			        					      	         
+					dweet.find('.ui-dialog-titlebar-close').click(function() {
+						let id = $(this.parentElement).attr('id');
+						$("#"+id).remove();
+					});
+					dweet.draggable({
+					scroll: false
+					, handle: '.ui-dialog-titlebar'
+					, containment: 'body'
+					, start: function() {
+						if (!!$(this).css('bottom')) {
+							$(this).css('bottom', '').css(Settings.isRtl ? 'left' : 'right', '');
+						}
+					}
+					, drag: function() {
+						if ($(this).position().x + $(this).width() >= $(this).parent().width()) {
+							return false;
+						}
+					}
+				});
+				dweet.resizable({
+					minHeight: 140
+					, minWidth: 255
+				});		
+				dweet.find('.update-btn').button().click(function() {
+					var id = $(this.parentElement.parentElement).attr('id');
```

Comment:
please use `let` or `const` instead of `var`

Reply:
Done